tests/autopilot-capacity/aut_test.go

```diff
@@ -184,18 +185,27 @@ var _ = Describe(fmt.Sprintf("{%sWaitForWorkload}", testNameSuite), func() {
 			})
 		}
 	})
+	JustAfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
+			CollectSupport()
+			DescribeNamespace(contexts)
+		}
+	})
 })
```

Comment:
is it possible to put these 6 lines into a afterEachTest function in common.go and just invoke that from all tests. Looks like all 6 lines are duplicated in each test